recipes/paragraph/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "paragraph" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: python-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5507561d8c5659047e16eb2e9cded1d9a4c1fc88489dd2bc09b76e660381c363
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
```

Comment:
```suggestion
    - pip
    - setuptools_scm
```

Reply:
Renamed the folder to `python-paragraph` as requested.